src/control/nav2_controller_dwb/CMakeLists.txt

```diff
@@ -17,9 +17,11 @@ find_package(nav2_tasks REQUIRED)
 find_package(nav2_robot REQUIRED)
 find_package(nav2_planning_msgs REQUIRED)
 find_package(dwb_core REQUIRED)
+find_package(Boost REQUIRED COMPONENTS system thread)
```

Comment:
I'm going to remove the Boost dependencies in my next PR for costmap_2d, so we should be able to remove this soon.